src/app/containers/ATIAnalytics/params/media/buildParams.test.js

```diff
@@ -1,8 +1,14 @@
 import { buildMediaATIParams, buildMediaATIUrl } from './buildParams';
-import { buildATIPageTrackPath } from '../../atiUrl';
 
-jest.mock('../../../../lib/analyticsUtils');
-jest.mock('../../atiUrl');
+jest.mock('../../../../lib/analyticsUtils', () => {
```

Comment:
do we still need this mock?

Reply:
Yes, only the `getPublishedDatetime` and `getCurrentTime` fns within it are mocked, so their outputs can be consistent, making tests predictable.

Reply:
Are the outputs of these functions deterministic? https://github.com/bbc/simorgh/pull/3559/files#diff-969e14df75b9da2e46ff6540eabc15a0R26-R29

If so I don't think they need to be mocked.

Reply:
You're right. It's been fixed